golangci-linters.yaml

```diff
@@ -65,7 +65,7 @@ linters:
     - noctx
     - nolintlint
     - nonamedreturns
-    - nosnakecase
+    # - nosnakecase
```

Comment:
Remove?

Reply:
The template kept these as commented out. I think this is for convenience to be able to enable or disable them easily.